storey/table.py

```diff
@@ -1075,6 +1090,9 @@ def __init__(self, max_value=None, set_data=None, set_time=None):
     def aggregate(self, time, value):
         self._set_value(self.value + value * value)
 
+    def aggregate_lua_script(self, vl1, vl2):
+        return f'{vl1}+{vl2}'
```

Comment:
Hm, this doesn't look like the right implementation for sum of squares. Unless I'm missing something, we should add a test that catches this.

Reply:
the expected Sqr math function (x*x) is done in the aggregate function.
the aggregate_lua_script() is used as component for building the lua script. It is called with vl1 containing the old value and vl2 is the square of the value to be added.
so there is no issue here (and there are tests covering this functionality)